api/interactives.go

```diff
@@ -363,6 +319,76 @@ func (api *API) UpdateInteractiveHandler(w http.ResponseWriter, req *http.Reques
 	WriteJSONBody(interactive, w, http.StatusOK)
 }
 
+func (api *API) PatchInteractiveHandler(w http.ResponseWriter, req *http.Request) {
```

Comment:
minor global : (prob in client also) ..maybe name the end point to reflect that it only updates file (not mdata)

Reply:
i dont think we would have a patch endpoint per update but instead driven from the `PatchRequest.Action` attribute - but in saying this its obv not clear 🤠 ....thinking about it how about I change this model slightly (and our mongo!) 😄 

- move import_success and import_message - inside the archive col
- update PatchRequest.Action -> PatchRequest.Attribute - then you know which attribute the Patch is replacing?

I think makes more sense tbh - wha ya reckon?

Reply:
Sounds good .. as long as the end user is prevented from attempting say a metadata update (title/label) using this patch EP..